src/main/resources/XWiki/MapMacro.xml

```diff
@@ -676,6 +692,13 @@ No content.
 {{/code}}
 
 {{map location="Rue des Thermopyles, 75014 Paris" width="400" height="300"}}A beautiful street in Paris{{/map}}
+
+== Map using geographic coordinate ==
+{{code}}
```

Comment:
No. You could replace {{code}}<content>{{/code}} by ##<content>## (unfortunately ##{{{<content>}}}### doesn't work).

Again that's only if you don't need the code macro in your macro and you only need it in the example; it would be a pity to mandate a dep on the code macro just for this. You could also declare the dep on the code macro as a provided scope but your code won't render fine if the code macro is not installed.